main.js

```diff
@@ -282,13 +297,18 @@ function sortedPrintObject(obj, indentCount) {
 
 function main() {
     if (process.argv.length < 3) {
-        console.log("Please pass a valid path. Usage: node main.js /path/to/winjs.d.ts");
+        console.log("Please pass a valid paths. Usage: node main.js /path/to/winjs.d.ts [/path/to/ExtensionXYZ.d.ts]");
```

Comment:
Can you update your addition to include a `...` like this:

`[/path/to/ExtensionXYZ.d.ts ...]`